Gopkg.lock

```diff
@@ -399,15 +386,6 @@
   packages = ["."]
   revision = "2788f0dbd16903de03cb8186e5c7d97b69ad387b"
 
-[[projects]]
-  name = "github.com/machine-drivers/docker-machine-driver-hyperkit"
-  packages = [
-    "pkg/drivers",
-    "pkg/hyperkit"
-  ]
-  revision = "88bae774eacefa283ef549f6ea6bc202d97ca07a"
-  version = "v1.0.0"
```

Comment:
@praveenkumar why are we removing this?

Reply:
@LalatenduMohanty not needed anymore since we are adding directly the driver struct to codebase same as we add for xhyve and others.